nri-config-generator/go.mod

```diff
@@ -1,14 +1,34 @@
 module github.com/newrelic/nri-config-generator
 
-go 1.16
+go 1.18
 
 require (
-	github.com/golangci/golangci-lint v1.40.1
 	github.com/mitchellh/mapstructure v1.1.2
 	github.com/newrelic/infra-integrations-sdk/v4 v4.1.0
 	github.com/pkg/errors v0.9.1
 	github.com/sirupsen/logrus v1.8.1
 	github.com/spf13/viper v1.7.1
 	github.com/stretchr/testify v1.7.0
```

Comment:
just the same comment on bumping up dependencies here.

Reply:
done